features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql

```diff
@@ -163,6 +163,33 @@ CREATE TABLE IF NOT EXISTS IDN_OAUTH2_AUTHZ_CODE_SCOPE(
              FOREIGN KEY (CODE_ID) REFERENCES IDN_OAUTH2_AUTHORIZATION_CODE (CODE_ID) ON DELETE CASCADE)
 /
 
+CREATE TABLE IF NOT EXISTS IDN_OAUTH2_DEVICE_FLOW (
+	    CODE_ID VARCHAR(255),
+        DEVICE_CODE VARCHAR(2048),
```

Comment:
Do we need 2048 length? it may cause an issue in some DB, as this is included as the unique index.

Reply:
Fixed